### PR TITLE
Add mdfp entry for msnbc.com, newsvine.com

### LINKED ISSUE #1293: cdn.lib.newsvine.com breaks video
http://www.msnbc.com/morning-joe/watch/gop-rep-trump-taking-advice-from-swamp-creatures-911153731866

`cdn.lib.newsvine.com` breaks video. Move from red to yellow.

### PR DESCRIPTION
**Pull request by eenblam**
Fixes #1293. Follows up on and closes #1570.

### CODE REVIEW

**Comment by eenblam** [comment]
Firefox beta build is failing due to the following Selenium test timing out:

```python
def test_should_not_detect_ls_of_third_party_script(self):
    # a third-party script included by the top page (not a 3rd party frame)
    self.load_url("https://rawgit.com/gunesacar/b366e3b03231dbee9709fe0a614faf10/raw/48e02456aa257e272092b398772a712391cf8b11/localstorage_from_third_party_script.html", wait_on_site=5)
    self.assertFalse(self.detected_tracking_by("githack.com"))
```

This seems to be an issue caused by either:
1. the network/Travis. Can we retry the build?
2. `SuperCookieTest.detected_tracking_by` checking the snitch map. I'm not sure how the requested change would create such a drastic hit to performance, but I'm looking into it presently.

File: src/js/multiDomainFirstParties.js [hunk 1]
```diff
@@ -126,6 +126,7 @@ var _multiDomainFirstPartiesArray = [
   ],
   ["mobilism.org.in", "mobilism.org"],
   ["morganstanley.com", "morganstanleyclientserv.com", "stockplanconnect.com", "ms.com"],
+	["msnbc.com", "newsvine.com"],
```
Comment (cowlicks) [inline src/js/multiDomainFirstParties.js]: I think this is indented with a tab, can you replace it with 2 spaces?

Comment (eenblam) [inline src/js/multiDomainFirstParties.js]: Yeah, my bad. Accidentally had `shiftwidth` and `tabstop` set in Vim without `expandtab`, so it looked right on my end despite the difference.

**Comment by cowlicks** [comment]
Thanks @eenblam !

**Pull request merged**